steps/http/session.go

```diff
@@ -187,3 +193,11 @@ func (s *Session) ValidateResponseBodyEmpty(ctx context.Context) error {
 	}
 	return fmt.Errorf("The response body is not empty")
 }
+
+// StoreResponseBodyJSONPropertyValue stores in the context a key value received in the HTTP Response's body.
```

Comment:
I'd rename the method to `StoreResponseBodyJSONPropertyInContext`.
I'd also rephrase the comment to: `StoreResponseBodyJSONPropertyInContext extracts a JSON property from the HTTP response body and stores it in the context.`

Reply:
Done